test/test_helper.exs

```diff
@@ -1 +1,37 @@
+defmodule TestOAuthToken do
```

Comment:
This just mimics the struct/type defs from the dependency's `AccessToken` module